administrator/language/en-GB/en-GB.mod_quickicon.ini

```diff
@@ -22,6 +22,7 @@ MOD_QUICKICON_EXTENSION_MANAGER="Extension Manager"
 MOD_QUICKICON_GLOBAL_CONFIGURATION="Global Configuration"
 MOD_QUICKICON_GROUP_DESC="The group of this module (this value is compared with the group value used in <strong>Quick Icons</strong> plugins to inject icons). The 'mod_quickicon' group always displays the Joomla! core icons."
 MOD_QUICKICON_GROUP_LABEL="Group"
+MOD_QUICKICON_HEADER_ICON_LABEL="Header icon"
```

Comment:
As it is a label it should follow the style guise and both words should be capitalised